wordpress-mobile/WordPress-Android
# [Gutenberg][TEST][DO NOT MERGE] Test `react-native-prompt-android` package integration

**NOTE: This PR is meant to be only for testing so it shouldn't be merged.**

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3548

_This PR includes the changes from https://github.com/wordpress-mobile/WordPress-Android/pull/14623._

To test:
1. Open a post/page
2. Observe that the app doesn't crash

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
